lib/matplotlib/pyplot.py

```diff
@@ -895,24 +965,24 @@ def gcf():
         return figure()
 
 
-def fignum_exists(num):
+def fignum_exists(num: int) -> bool:
     """Return whether the figure with the given id exists."""
     return _pylab_helpers.Gcf.has_fignum(num) or num in get_figlabels()
 
 
-def get_fignums():
+def get_fignums() -> list[int]:
     """Return a list of existing figure numbers."""
     return sorted(_pylab_helpers.Gcf.figs)
 
 
-def get_figlabels():
+def get_figlabels() -> list[Any]:
     """Return a list of existing figure labels."""
     managers = _pylab_helpers.Gcf.get_all_fig_managers()
     managers.sort(key=lambda m: m.num)
     return [m.canvas.figure.get_label() for m in managers]
 
 
-def get_current_fig_manager():
+def get_current_fig_manager() -> FigureManagerBase | None:
```

Comment:
In the future we should probably tighten this up eventually, as while the base `Canvas` may have `obj.manager` as `None`, but if it is coming out of `gcf()` then it is a figure that pyplot is managing and thus we know that manager must not be `None`.

Is this easy to express?  I think this may be a good use of `cast(FigureManagerBase, ret)`?